internal/fsutil/create_test.go

```diff
@@ -62,98 +68,101 @@ var createTests = []createTest{{
 		"/tmp/": "dir 01775",
 	},
 }, {
+	summary: "Cannot create a subdirectory without `MakeParents` set",
```

Comment:
```suggestion
	summary: "Cannot create a parent directory without MakeParents set",
```
Note: We do not use ``Word`` normally in Go, either the identifier is included normally or you can use `[..]` (see docstring documentation).

Reply:
Fixed in [4090b4c](https://github.com/letFunny/chisel/pull/17/commits/4090b4c4c27eda1bd386bb1757572a4529887ad9).